plugins/cpython.py

```diff
@@ -0,0 +1,98 @@
+from plugins.api import Plugin
+from incparser.annotation import Annotation, Footnote, Heatmap, ToolTip
+
+import os
+import os.path
+import subprocess
+import tempfile
+
+class CPythonFuncProfileMsg(Annotation):
+    def __init__(self, annotation):
+        self._hints = [ToolTip(), Footnote()]
+        super(CPythonFuncProfileMsg, self).__init__(annotation)
+
+    def get_hints(self):
+        return self._hints
+
+
+class CPythonFuncProfileVal(Annotation):
+    def __init__(self, annotation):
+        self._hints = [Heatmap()]
+        super(CPythonFuncProfileVal, self).__init__(annotation)
+
+    def get_hints(self):
+        return self._hints
+
+
+class CPythonProfiler(Plugin):
+    """CPython 2 prfofiler.
+    Requires the Python standard library.
+    """
+
+    def __init__(self):
+        super(CPythonProfiler, self).__init__("Python 2.7.5",
+                                              "CPython profiler")
+
+    def run_tool(self):
+        f = tempfile.mkstemp()
+        self.export(f[1])
+
+        # python -m cProfile [-o output_file] [-s sort_order] myscript.py
+        proc = subprocess.Popen(["python2",
```

Comment:
I *think* (but I might be wrong) that `python2.7` is a safer executable name (I have `python2.7` but not `python2` on my machine).